reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md

```diff
@@ -17,23 +17,19 @@ Explains how to resolve problems with scheduled jobs
 
 # LONG DESCRIPTION
 
-This section describes some of the problems that you might encounter when
-using the scheduled job features of Windows PowerShell and it suggests
-solutions to these problems.
+This section describes some of the problems that you might encounter when using the scheduled job features of Windows PowerShell and it suggests solutions to these problems.
```

Comment:
Do not change the line breaks. About topics have specific formatting requirements.